docs/blueprints/EP000.rst

```diff
@@ -0,0 +1,153 @@
+:EP: 00
+:Title: Enhancement Proposals
+:Authors:
+    - Antonio Francisco <ajoaoff@gmail.com>
+    - Carlos Magno <cmagnobarbosa@gmail.com>
+    - Gleyberson Andrade <gleybersonandrade@gmail.com>
+    - Humberto Diógenes <hdiogenes@gmail.com>
+    - José Mauro Ribeiro <zemauror@gmail.com>
+    - Rogério Motitsuki <rogerio@ansp.br>
+:Created: 2020-05-14
+:Kytos-Version: 2020.1
+:Status: Draft
+:Type: Standards Track
+
+
+########
+Abstract
+########
+This blueprint is proposed to clarify what a blueprint is, its purpose, list the types of blueprints, explain which sections should be included in a blueprint, the information that should be included in each topic and the workflow involving a blueprint proposal. Therefore, this blueprint is intended to work as a meta blueprint.
+
+##########
+Motivation
+##########
+Kytos Project uses the word 'blueprint' to refer to a very specific type of document that plays a very important role in the development process. Anyone who is not familiar with the concept should be able to understand and start a draft blueprint using the EP0 (blueprint 00).
+
```

Comment:
IMO, we need to improve this motivation. What are the problems of not having blueprints? The motivation should be "why are we using blueprints after all".

Reply:
Well, thanks for your comments. I updated the section "Motivation", please let me know if you have any suggestion.